x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -193,7 +194,8 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
           title = i18n.OPENED_ALERT_FAILED_TOAST;
           break;
         case 'in-progress':
```

Comment:
Can you create a separate issue to remove this later on?

Reply:
issue here https://github.com/elastic/kibana/issues/109030